llvm/include/llvm/Transforms/IPO/ProfiledCallGraph.h

```diff
@@ -85,7 +86,7 @@ class ProfiledCallGraph {
     std::queue<ContextTrieNode *> Queue;
     for (auto &Child : ContextTracker.getRootContext().getAllChildContext()) {
       ContextTrieNode *Callee = &Child.second;
-      addProfiledFunction(ContextTracker.getFuncNameFor(Callee));
+      addProfiledFunction(Callee->getFuncName());
```

Comment:
ProfiledCallGraph needs to build graph with actual function name. This is because it is used to build up processing order for SampleProfileLoader::SymbolMap, which is keyed on string name. 

This change would break it as it changes ProfiledCallGraph to also use MD5. Same for similar changes below.

Reply:
now obsolete